lib/publiccloud/gce.pm

```diff
@@ -81,8 +82,20 @@ sub upload_img {
     return $img_name;
 }
 
-sub cleanup {
-    record_info('info', 'to be implemented');
+sub ipa {
+    my ($self, %args) = @_;
+
+    $args{credentials_file} = CREDENTIALS_FILE;
+    $args{instance_type}        //= 'n1-standard-2';
+    $args{cleanup}              //= 1;
```

Comment:
Maybe we should only set the defaults which really belongs to gce like credentials_file, instance_type, provider and user.

Reply:
makes sense, same for the other providers